src/cogs/voice_recording.py

```
@@ -0,0 +1,319 @@
+            if guild_id in self.voice_recorders:
+                recorder = self.voice_recorders[guild_id]
+                if recorder.is_connected:
+                    try:
+                        await recorder.leave_voice_channel()
+                        logger.info(f"機器人被斷開連接，自動清理連接狀態 (Guild: {guild_id})")
+                    except Exception as e:
+                        logger.error(f"自動清理連接狀態失敗: {e}")
+
+
+# 註冊 Cog
+async def setup(bot: commands.Bot) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)